frontend/src/components/navbar.js

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react'
+
+import Login from './LoginForm'
+import userContext from '../contexts/context'
+
+function Navbar() {
+  let user = useContext(userContext)
+    
+    if (user.admin) {
+      return(
```

Comment:
Have a single `return` and do inside `{user.admin && <div>...</div>}`

Reply:
I used a different syntax, tell me if it works for you

 {user.admin ? (<div>...</div>) : (<div>...</div>)}